.circleci/config.yml

```diff
@@ -136,6 +136,9 @@ jobs:
       - run:
           name: Checkout UI project
           command: git clone https://github.com/ministryofjustice/send-legal-mail-to-prisons.git
+      - run:
+          name: Install curl
+          command: apt-get update && apt-get install -y curl
```

Comment:
Added this as smoke tests were failing when running on dev - https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-send-legal-mail-staff-ui/96